Filter.cpp:421:35: error: 'class QStringList'

/home/github/qtermwidget/lib/Filter.cpp: In member function 'Konsole::UrlFilter::HotSpot::UrlType Konsole::UrlFilter::HotSpot::urlType() const':
/home/github/qtermwidget/lib/Filter.cpp:421:35: error: 'class QStringList' has no member named 'constFirst'
     QString url = capturedTexts().constFirst();

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,6 +7,7 @@ include(GenerateExportHeader)
 include(CMakePackageConfigHelpers)
 include(CheckFunctionExists)
 
+set(REQUIRED_QT_VERSION "5.6")
 set(LXQTBT_MINIMUM_VERSION "0.3.0")
 
 option(UPDATE_TRANSLATIONS "Update source translation translations/*.ts files" OFF)
@@ -27,8 +28,8 @@ endif()
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
-find_package(Qt5Widgets REQUIRED)
-find_package(Qt5LinguistTools REQUIRED)
+find_package(Qt5Widgets "${REQUIRED_QT_VERSION}" REQUIRED)
+find_package(Qt5LinguistTools "${REQUIRED_QT_VERSION}" REQUIRED)
 find_package(lxqt-build-tools ${LXQTBT_MINIMUM_VERSION} REQUIRED)
 include(LXQtTranslateTs)
 include(LXQtCompilerSettings NO_POLICY_SCOPE)

--- a/README.md
+++ b/README.md
@@ -15,8 +15,8 @@ This project is licensed under the terms of the [GPLv2](https://www.gnu.org/lice
 
 ### Compiling sources
 
-The only runtime dependency is qtbase ≥ 5.4.   
-In order to build CMake ≥ 3.0 is needed as well as optionally Git to pull latest VCS checkouts.
+The only runtime dependency is qtbase ≥ 5.6.
+In order to build CMake ≥ 3.0.2 and [lxqt-build-tools](https://github.com/lxde/lxqt-build-tools/) >= 0.3 are needed as well as Git to pull translations and optionally latest VCS checkouts.
 
 Code configuration is handled by CMake. Building out of source is strongly recommended. CMake variable `CMAKE_INSTALL_PREFIX` will normally have to be set to `/usr`, depending on the way library paths are dealt with on 64bit systems variables like `CMAKE_INSTALL_LIBDIR` may have to be set as well.   
 

